docs/tables.md

```diff
@@ -0,0 +1,458 @@
+{
+    "fractal_table_version": "1",
+    "type": "masking_roi_table",
+    "region": { "path": "../labels/label_DAPI" },
+    "instance_key": "label",
+    "encoding-type": "anndata",
+    "encoding-version": "0.1.0",
+}
+```
+
+**Table columns**
+
+On top of the required ROI-table colums, a masking ROI table must include the
+table which is defined in its `instance_key` attribute, e.g. the `label` one in
+the example above.
+
+### Feature tables
+
+**Motivation**
+
+The typical use case for feature tables is to store measurements related to
+segmented objects, while mantaining a link to the original instances (e.g.
+labels). Note that the current specification is aligned to the one of [masking
+ROI tables](#masking-roi-tables), since they share the same kind of use case,
+but the two may diverge in the future.
+
+As part of the current `fractal-tasks-core` tasks, measurements can be
+performed e.g. via `regionprops` from `scikit-image`, as wrapped in
+[napari-skimage-regionprops](https://github.com/haesleinhuepf/napari-skimage-regionprops)).
+
+**Zarr attributes**
+
+For this kind of tables, `fractal-tasks-core` closely follows the [proposed
+NGFF update mentioned above](https://github.com/ome/ngff/pull/64). The
+requirements on the Zarr attributes of a given table are:
+
+* Attributes must contain a `type` key, with value `feature_table`[^2].
+* Attributes must contain a `region` key; the corresponding value must be an
+  object with a `path` key and a string value (i.e. the path to the data the
+  table is annotating).
+* Attributes must include a key `instance_key`, which is the key in `obs` that
+  denotes which instance in `region` the row corresponds to.
+
+Here is an example of valid Zarr attributes
+```json title="image.zarr/tables/table_1/.zattrs" hl_lines="3 4 5"
+{
+    "fractal_table_version": "1",
+    "type": "feature_table",
+    "region": { "path": "../labels/label_DAPI" },
+    "instance_key": "label",
+    "encoding-type": "anndata",
+    "encoding-version": "0.1.0",
+}
+```
+
+**Table columns**
+
+A feature table must include the table which is defined in its `instance_key`
+attribute, e.g. the `label` one in the example above.
+
+## Examples
+
+### Use cases for ROI tables
+
+#### OME-Zarr creation
+
+OME-Zarrs created via `fractal-tasks-core` (e.g. by parsing Yokogawa images via
+the
+[`create_ome_zarr`](../reference/fractal_tasks_core/tasks/create_ome_zarr/#fractal_tasks_core.tasks.create_ome_zarr.create_ome_zarr)
+or
+[`create_ome_zarr_multiplex`](../reference/fractal_tasks_core/tasks/create_ome_zarr_multiplex/#fractal_tasks_core.tasks.create_ome_zarr_multiplex.create_ome_zarr_multiplex)
+tasks) always include two specific ROI tables:
+
+* The table named `well_ROI_table`, which covers the NGFF image corresponding to the whole well[^1];
+* The table named `FOV_ROI_table`, which lists all original FOVs.
+
+Each one of these two tables includes ROIs that are only defined in the XY
```

Comment:
Still not clear about this. They are defined in Z, just as spanning the whole Z range, no?

Reply:
The spec should be that they define the XYZ extent of those FOVs (and the well and FOV always span the whole Z range). But as far as the ROI table is concerned, it only knows that because we put in the correct len_z_micrometer)

Whether our code contains an assumption somewhere that those pixel sizes in Z would always be 1: to be verified, but that would be a bug